text_extensions_for_pandas/array/test_tensor.py

```diff
@@ -299,11 +313,14 @@ def test_sum(self):
             repr(result2_df),
             textwrap.dedent(
                 """\
-                                value
-                key                  
-                a    [[2, 2], [2, 2]]
-                b    [[1, 1], [1, 1]]
-                c    [[3, 3], [3, 3]]"""
+                             value
+                key               
+                a   [[2 2]
+                 [2 2]]
+                b   [[1 1]
+                 [1 1]]
+                c   [[3 3]
+                 [3 3]]"""
```

Comment:
The change in `_from_sequence` caused this. This looks worse, but `_from_sequence` was wrong before so we can figure out how to format this better later.

Reply:
ok